Rudder publishing (#260)
avohq/docs: pages/workspace/tracking-plan/publishing.mdx

Avo will not impact any events that are only defined in Lexicon, or events that have not been added to the Lexicon integration in Avo.

<img
  src={require('../../../images/publishing_rudderstack.svg')}
  alt="Amplitude"
  style={{ margin: '32px 0 -32px 0' }}
/>

> ## <a name="rudderstack"></a> RudderStack

Reviewer: ```suggestion
## <a name="rudderstack"></a> RudderStack Tracking Plans
```

> The integration to [RudderStack](https://rudderstack.com/blog/dogfooding-at-rudderstack-tracking-plans-part-1/) allows for your Tracking Plan to be published into RudderStack whenever branches are merged in your workspace or with a push of a button.

Reviewer: Hemingway adjustments:

The Rudderstack integration enables you to publish your Avo Tracking Plan into RudderStack. You can configure the integration to publish whenever a branch is merged or with a push of a button.

### Configure the RudderStack integration

> To enable the direct integration with Segment Protocols, you need to configure it with the following parameters from RudderStack.

Reviewer: ```suggestion
To enable the direct integration with RudderStack Tracking Plans, you need to configure it with the following parameters from RudderStack.
```

- **RudderStack Account Email Address**: The email of a RudderStack account with Read-Write access
- **RudderStack Tracking Plan Name**: The display name of the RudderStack Tracking Plan you want to publish to
- **RudderStack Access Token**: [Create new Personal Access Token](https://rudderstack.com/docs/transformations/api-access-token/#access-token) for the account with the email address you provided above

### Publishing to RudderStack

Avo will override all events in the RudderStack Tracking Plan that are being imported from Avo. Any changes made to an event in RudderStack Tracking Plan between Avo publish will be overridden on next Avo publish.

> Avo will not override any events that are only defined in RudderStack, or events that have not been added to the RudderStack integration in Avo.

Reviewer: ```suggestion
Avo will not override any events that are only defined in RudderStack, or events that have not been excluded from the RudderStack integration in Avo with filters.
```
